mlir/test/Integration/GPU/CUDA/dump-ptx.mlir

```diff
@@ -11,3 +11,14 @@ gpu.module @bar {
     llvm.return
   }
 }
+
+// CHECK-LABEL: Generated by LLVM NVPTX Back-End
```

Comment:
Can you please also check for `// CHECK: .visible .func fma()`?

Reply:
done.